app/components/owners-list.hbs

```diff
@@ -11,8 +11,8 @@
         local-class="link"
         data-test-owner-link={{owner.login}}
       >
-        <UserAvatar @user={{owner}} @size="medium-small" local-class="avatar" />
-        <span local-class="name">{{or owner.display_name owner.name}}</span>
+        <UserAvatar @user={{owner}} @size="medium-small" local-class="avatar" aria-hidden="true" />
+        <span local-class="name {{unless this.showDetailedList "hidden-name"}}">{{or owner.display_name owner.name owner.login}}</span>
```

Comment:
I feel like we could move the `or` into a getter in the model, but that's just a nit.

Reply:
yeah, I had a similar though, but I think I'd rather extract a `get display_name()` on the `User` model instead